morpheus/_lib/stages/module.cpp

```diff
@@ -15,6 +15,13 @@
  * limitations under the License.
  */
 
+#include <mrc/segment/builder.hpp>  // for Builder
+#include <mrc/segment/object.hpp>
+#include <mrc/utils/string_utils.hpp>
+#include <pybind11/attr.h>      // for multiple_inheritance
+#include <pybind11/pybind11.h>  // for arg, init, class_, module_, str_attr_accessor, PYBIND11_MODULE, pybind11
+#include <pybind11/pytypes.h>   // for dict, sequence
+// for pathlib.Path -> std::filesystem::path conversions
```

Comment:
Delete this line so the formatter can rearrange the includes properly. It wont break any comment boundaries